docs/sphinx/packages/ant.rst

```diff
@@ -16,8 +16,11 @@ The :program:`ant` build tool for Java.
 +------------------+------------------------------+
 | Homebrew         | ant                          |
 +------------------+------------------------------+
-| RedHat/CentOS    | N/A                          |
+| RedHat/CentOS    | ant*                         |
```

Comment:
Which component has a dependency on the `ant` package? I would have expected this package no longer to be a requirement since `ome-model` was decoupled out of Bio-Formats.

Reply:
None, it's there for historical reasons only.